### DISCUSSION TITLE
Windows Compiling & Linking

### DISCUSSION THREAD

**Discussion by Asadaaaaa** [ArthurSonzogni/FTXUI#269] [Q&A] [answered]
Is there any example source code and example for compiling & linking with ftxui on Windows OS?

**Comment by ArthurSonzogni** [comment, marked as answer]
What about trying:
https://github.com/ArthurSonzogni/ftxui-starter

I don't know which compiler you are using on Windows. Is it visual studio? Then, I believe it can work by "double-clicking" on the CMakeLists.txt while inside?
